Loop/Managers/LoopDataManager.swift

```diff
@@ -1826,6 +1833,7 @@ extension LoopDataManager {
                         maximumBolus: settings.maximumBolus,
                         suspendThreshold: settings.suspendThreshold,
                         insulinSensitivitySchedule: insulinSensitivitySchedule,
-                        carbRatioSchedule: carbRatioSchedule)
+                        carbRatioSchedule: carbRatioSchedule,
+                        glucoseUnit: glucoseUnit)
```

Comment:
See my [comment](https://github.com/tidepool-org/LoopKit/pull/142#discussion_r455329048) in https://github.com/tidepool-org/LoopKit/pull/142